### PR TITLE
Content fields parsing

### LINKED ISSUE #1603: The output of the command `./pyrsia inspect-log docker --help` is incorrect
### Steps taken/Steps to reproduce

Run `./pyrsia inspect-log docker --help`, the output is as follows:
```
xuefeihan@xuefeih-mac pyrsia % ./target/debug/pyrsia inspect-log docker --help                                       
Show transparency logs for a Docker image

Usage: pyrsia inspect-log docker [OPTIONS] --image <IMAGE>

Options:
      --image <IMAGE>    The docker image (e.g. alpine:3.15.3 or alpine@sha256:1e014f84205d569a5cc3be4e108ca614055f7e21d11928946113ab3f36054801
      --format <FORMAT>  The output format [default: json] [possible values: json, csv]
      --fields <FIELDS>  The list of output fields separated by comma ','.
                         Available fields:
                         	- field: 'Id',	description: TransparencyLog record identity
                         	- field: 'PackageType',	description: Package type (maven, docker and so on)
                         	- field: 'PackageSpecificId',	description: Package specific identity
                         	- field: 'NumArtifacts',	description: Number of artifacts
                         	- field: 'PackageSpecificArtifactId',	description: Package specific artifact identity
                         	- field: 'ArtifactHash',	description: Artifact hash
                         	- field: 'SourceHash',	description: Source hash
                         	- field: 'ArtifactId',	description: Artifact identity
                         	- field: 'SourceId',	description: Source identity
                         	- field: 'Timestamp',	description: Timestamp
                         	- field: 'Operation',	description: Operation (AddArtifact, RemoveArtifact, AddNode, RemoveNode)
                         	- field: 'NodeId',	description: Peer node identity
                         	- field: 'NodePublicKey',	description: Node public key
                         
  -h, --help             Print help
```

### Expected Behavior

1. `--image <IMAGE>` should not be included in `Options`
2. In `-field: 'Id'`: `Id` is an invalid value, the correct value may be `id`
3. In `-field: 'PackageType'`: `PackageType` is an invalid value
4. In `-field: 'PackageSpecificId'`: `PackageSpecificId` is an invalid value
5. In `-field: 'NumArtifacts'`: `NumArtifacts` is an invalid value
6. In `-field: 'PackageSpecificArtifactId'`: `PackageSpecificArtifactId` is an invalid value
7. In `-field: 'ArtifactHash'`: `ArtifactHash` is an invalid value
8. In `-field: 'SourceHash'`: `SourceHash` is an invalid value
9. In `-field: 'ArtifactId'`: `ArtifactId` is an invalid value
10. In `-field: 'SourceId'`: `SourceId` is an invalid value
11. In `-field: 'Timestamp'`: `Timestamp` is an invalid value, the correct value may be `timestamp`
12. In `-field: 'Operation'`: `Operation` is an invalid value
13. In `-field: 'NodeId'`: `NodeId` is an invalid value
14. In `-field: 'NodePublicKey'`: `NodePublicKey` is an invalid value

### Experienced Instance of the bug

Rust version: rustc 1.66.1 (90743e729 2023-01-10)
OS: Mac OS
Pyrsia version: commit b0116a3a94b57d14d01257d6ad61105aee63e05e

### PR DESCRIPTION
**Pull request by dgarus**
## Description

Fixes #1603, #1604

- fixed `--fields` help description
- field names are made case-insensitive
- fixed fields parsing error output

## PR Checklist

- [x] I've read the [contributing guidelines](https://github.com/pyrsia/.github/blob/main/contributing.md).
- [x] I've read ["What is a Good PR?"](https://github.com/pyrsia/pyrsia/blob/main/docs/community/get_involved/good_pr.md)
- [x] I've included a good title and brief description along with how to review them.
- [x] I've linked any associated an [issue](https://github.com/pyrsia/pyrsia/issues).

### Code Contributions

- [x] I've run [pre-commit.sh](https://github.com/pyrsia/pyrsia/blob/main/pre-commit.sh) ([pre-commit.bat](https://github.com/pyrsia/pyrsia/blob/main/pre-commit.bat)) successfully.
- [x] I've run the [integrations tests](https://github.com/pyrsia/pyrsia-integration-tests#how-to-set-up-and-run-the-tests)
on the PR branch and all tests passes.

### CODE REVIEW

**Review by fishseabowl — DISMISSED** [review]
/lgtm. A suggestion: when parsing parameters fails, it is better to add help information, like this:
```
xuefeihan@xuefeih-mac pyrsia % ./target/debug/pyrsia inspect-log docker --fields packag --image alpine:3.16 
Parsing of arguments failed with error: ParseTransparencyLogFieldError { invalid_field: "packag" }


For more information, try '--help'.
```

**Comment by dgarus** [comment]
@fishseabowl
Thank you for the review.
Fixed your comment, please have a look.

**Review by fishseabowl — DISMISSED** [review]
lgtm

File: src/node_api/model/request.rs [hunk 1]
```diff
@@ -181,7 +181,8 @@ impl FromStr for TransparencyLogField {
     type Err = ParseTransparencyLogFieldError;
 
     fn from_str(s: &str) -> Result<Self, Self::Err> {
-        let res = match s {
+        let field_name = s.to_lowercase();
+        let res = match field_name.as_str() {
```
Comment (dasmanas) [inline src/node_api/model/request.rs:185]: can it be a single line statement?

**Review by dasmanas — APPROVED** [review]

File: src/node_api/model/request.rs (hunk 1 above)
Comment (dgarus) [reply src/node_api/model/request.rs]: Yes

**Review by fishseabowl — APPROVED** [review]

**Comment by dgarus** [comment]
@fishseabowl @dasmanas 
Thanks for the review

**Pull request merged**